xbmc/Application.cpp

```diff
@@ -1838,8 +1843,8 @@ int CApplication::Run()
   CFileItemList& playlist = CServiceBroker::GetAppParams()->GetPlaylist();
   if (playlist.Size() > 0)
   {
-    CServiceBroker::GetPlaylistPlayer().Add(0, playlist);
-    CServiceBroker::GetPlaylistPlayer().SetCurrentPlaylist(0);
```

Comment:
Only to make sure that's the "0" is not a mistake since begin of Kodi :thinking:.
Looked to the begin and also as "0", see https://github.com/xbmc/xbmc/blob/efe0dc9f65ac425d16a10d296c30765437a60a96/xbmc/xbmc.cpp#L136-L140
on all other places it was defined with `PLAYLIST_NONE`, `PLAYLIST_MUSIC` and `PLAYLIST_VIDEO`.

Not know where I can look find the old `SVN` code to confirm this with 0 was thought really as `PLAYLIST_MUSIC` and not as `PLAYLIST_NONE`.

Reply:
Yes this is a known issue regarding app params, I think there's a bug report already for that. Unrelated to this pr.